code/game/machinery/kitchen/processor.dm

```diff
@@ -29,6 +29,29 @@
 	for(var/obj/item/weapon/stock_parts/manipulator/M in component_parts)
 		rating_speed = M.rating
 
+/obj/machinery/processor/process()
+	..()
+	// The irony
+	// To be clear, if it's grinding, then it can't suck them up
+	if(processing)
+		return
+	var/mob/living/carbon/slime/picked_slime
+	for(var/mob/living/carbon/slime/slime in range(1,src))

```

Comment:
That's not how its meant to work, either. It's meant to work for the turf the processor is on, it's just meant to exclude slimes *inside* the processor.